docassemble/assemblylinewizard/data/sources/output_patterns.yml

```diff
@@ -0,0 +1,317 @@
+#########################################
+# Each dictionary entry is a template for a YAML block in the generated interview
+#########################################
+# Overall metadata
+# TODO: this could be a good place to also store things like the list of
+# courts, the types of forms, etc. Other things people might want to configure
+# about how the weaver works. OR: that could be a separate YAML file.
+#################################
+# All imports must be explicitly listed. They will not use Docassemble's usual missing variable
+# resolution, so you should make sure any text processing functions are safe to handle DAEmpty
+# objects.
+# You can provide local imports as a dictionary of lists. These will import from playground or runtime
+mako template local imports:
+  interview_generator: [fix_id, varname, mako_indent, using_string, escape_quotes, oneline]
+# To import from an installed package or third-party Python package, provide a list of strings
+#mako template imports: 
+#  - from docassemble.assemblylinewizard.interview_generator import fix_id, varname, mako_indent, using_string, escape_quotes, oneline
+# Ordinary question with or without fields
+question: |
+  % if event:
+  event: ${ event }
+  % endif
+  % if block_id:
+  id: ${ fix_id(block_id) }
+  % else:
+  id: ${ fix_id(question_text) }
+  % endif
+  question: |
+  ${ mako_indent(question_text,2) }
+  % if subquestion_text:
+  subquestion: |
+  ${ mako_indent(subquestion_text,2) }
+  % endif
+  % if field_list and len(field_list) > 1:
+  fields:
+  % for field in field_list:
+  ${ field.field_entry_yaml() }
+  % endfor
+  % else:
+  ${ fields[0].get_single_field_screen()}
```

Comment:
This should be `field_list[0].get_...`, and it'll cause errors if people only add one field.